app/src/main/java/com/rentminder/dto/Household.kt

```diff
@@ -1,6 +1,6 @@
 package com.rentminder.dto
 
-data class Household (var householdId : Int = 0, var houseName : String, var houseMembers : Int){
+data class Household (val householdId : Int = 0, val houseName : String, val houseMembers : Int){
```

Comment:
Simple improvements but necessary ones. Make sure to use vals instead of vars in data classes as it is good practice to use immutable properties of instances. URL: https://www.digitalocean.com/community/tutorials/kotlin-data-class

Reply:
Good.